cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -1,79 +1,54 @@
-import { fieldHelpPreview, labelByPosition, labelWidthSliderByName } from '../../locators';
+import { labelWidthSliderByName, labelByPositionNoIFrame } from '../../locators';
 import {
-  radioButtonByPosition,
   radioButtonFieldset,
   radioButtonLegend,
   radioButtonGroupNoIframe,
   radioButtonComponentByPositionNoIframe,
 } from '../../locators/radioButton/index';
 import { setSlidebar, positionOfElement } from '../helper';
-
-const INLINE = 'carbon-radio-button__help-text--inline';
-const RADIOBUTTON_INPUT_CSS = 'div:nth-child(1) input';
-
-Then('fieldHelpInline is enabled', () => {
-  fieldHelpPreview().should('have.class', INLINE);
-});
-
-Then('fieldHelpInline is disabled', () => {
-  fieldHelpPreview().should('not.have.class', INLINE);
-});
+import { FIELD_HELP_PREVIEW } from '../../locators/locators';
 
 Then('{string} radioButton on preview is {word}', (position, text) => {
-  labelByPosition(positionOfElement(position)).should('have.text', text);
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.text', text);
 });
 
-Then('{string} RadioButton has value {word}', (radioButtonName, text) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'value').should('contain', text);
+Then('{string} RadioButton has value {word}', (position, text) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('input').should('have.attr', 'value').should('contain', text);
 });
 
-Then('{string} RadioButton component is disabled', (radioButtonName) => {
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('be.disabled')
-    .and('have.attr', 'disabled');
+Then('{string} RadioButton component is disabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('have.attr', 'disabled');
 });
 
-Then('{string} RadioButton component is enabled', (radioButtonName) => {
-  cy.radioButtonComponent(radioButtonName).should('not.be.disabled').and('not.have.attr', 'disabled');
-  cy.radioButton(radioButtonName, RADIOBUTTON_INPUT_CSS).should('have.attr', 'type', 'radio')
-    .and('not.be.disabled')
-    .and('not.have.attr', 'disabled');
+Then('{string} RadioButton component is enabled', (position) => {
+  labelByPositionNoIFrame(positionOfElement(position)).should('not.have.attr', 'disabled');
 });
 
-Then('{string} RadioButton is set to reverse', (radioButtonName) => {
-  cy.radioButton(radioButtonName, 'div:nth-child(2) input').should('have.attr', 'role', 'radio');
+Then('{string} RadioButton is set to reverse', (position) => {
+  radioButtonComponentByPositionNoIframe(positionOfElement(position)).find('div:nth-child(2) input').should('have.attr', 'role', 'radio');
```

Comment:
here also move `.find(div:nth-child(2) input)` into the `index.js` file

Reply:
it's not a reusable locator, so I prefer to leave it as is